apps/src/p5lab/spritelab/SpritelabInput.jsx

```diff
@@ -4,14 +4,53 @@ import React from 'react';
 import {popPrompt} from './spritelabInputModule';
 import * as coreLibrary from './coreLibrary';
 
+const styles = {
+  container: {
+    background: 'rgba(34, 42, 51, 0.85)',
```

Comment:
is/should this be a color we reuse? if so, we should move this to our color constants file

Reply:
This color was specified by Mark in the spec, and it's darker than our darkest gray in the constants file. I think it's sort of one-off...

Reply:
ah okay, if these colors are custom for this component then it's probably fine to leave their definitions in this file